diff --git a/Beerhall/Program.cs b/Beerhall/Program.cs
index eab2bf1..db137c4 100644
--- a/Beerhall/Program.cs
+++ b/Beerhall/Program.cs
@@ -19,6 +19,9 @@ namespace Beerhall
 
         public static IWebHost BuildWebHost(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
+               .UseKestrel()
+               .UseUrls("http://*:5000")
+               .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseStartup<Startup>()
                 .Build();
     }
 
