diff --git a/src/DienstenCheques/Program.cs b/src/DienstenCheques/Program.cs
index eb51f23..0bc5c19 100644
--- a/src/DienstenCheques/Program.cs
+++ b/src/DienstenCheques/Program.cs
@@ -15,6 +15,7 @@ namespace DienstenCheques
                 .UseKestrel()
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseIISIntegration()
+		.UseUrls("http://*:5000")
                 .UseStartup<Startup>()
                 .Build();
 
